chain/network/src/peer_manager/tests/routing.rs

```diff
@@ -787,7 +787,7 @@ async fn max_num_peers_limit() {
     let chain = Arc::new(data::Chain::make(&mut clock, rng, 10));
 
     tracing::info!(target:"test", "start three nodes with max_num_peers=2");
-    let mut cfgs = make_configs(&chain, rng, 4, 4, true);
+    let mut cfgs = make_configs(&chain, rng, 4, 4, false);
```

Comment:
what does the last boolean mean? turning off auto connection?

Reply:
That's right, see here:
https://github.com/near/nearcore/blob/769b063721507664a5323b4b9c53b8c7fe9c0e02/chain/network/src/peer_manager/tests/routing.rs#L634-L640